soc/espressif/esp32c6/Kconfig

```diff
@@ -4,14 +4,15 @@
 config SOC_SERIES_ESP32C6
 	select RISCV
 	select RISCV_SOC_HAS_GP_RELATIVE_ADDRESSING
-	select DYNAMIC_INTERRUPTS
-	select CLOCK_CONTROL
-	select PINCTRL
+	select DYNAMIC_INTERRUPTS if SOC_ESP32C6_HPCORE
+	select CLOCK_CONTROL if SOC_ESP32C6_HPCORE
+	select PINCTRL if SOC_ESP32C6_HPCORE
 	select RISCV_ISA_RV32I
 	select RISCV_ISA_EXT_A
 	select RISCV_ISA_EXT_M
 	select RISCV_ISA_EXT_C
 	select RISCV_ISA_EXT_ZICSR
+	select RISCV_ISA_EXT_ZIFENCEI if SOC_ESP32C6_LPCORE
```

Comment:
ESP32C6 have zifencei extension.. Why filtering it out only to SOC_ESP32C6_LPCORE?